HIbernate/MailSenderTask2/src/main/java/com/demo/model/ServiceResponse.java

```diff
@@ -0,0 +1,34 @@
+package com.demo.model;
+
+public class ServiceResponse<T> {
```

Comment:
Why do you need this abstraction?
ResponseEntity has _HttpStatus status_ and _T body_